packages/react-composites/tests/browser/app/public/index.html

```diff
@@ -0,0 +1,16 @@
+<!-- Copyright (c) Microsoft Corporation. -->
+<!-- Licensed under the MIT license. -->
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="description" content="Chat Composite - Test App" />
+    <title>Chat Composite - Test App</title>
```

Comment:
nit: avoid `chat` references here as this looks generic and can be used for any integration testing

Reply:
Sure.